src/components/Layout/Header.tsx

```diff
@@ -53,55 +154,49 @@ const Header: FunctionComponent<Props> = props => {
-                                    Learn
-                                </NavDropdown.Item>
-                                <NavDropdown.Item
-                                    href="https://www.youtube.com/playlist?list=PL6zLuuRVa1_iDEP4EicZ8972RgyccCRGF"
-                                    target="_blank"
-                                    rel="noreferrer"
-                                >
-                                    Dev Tool Time
-                                </NavDropdown.Item>
-                                <NavDropdown.Item href="/podcast/">Sourcegraph Podcast</NavDropdown.Item>
-                                <NavDropdown.Item href="/case-studies">Case Studies</NavDropdown.Item>
-                            </NavDropdown>
-
-                            <NavDropdown id="useCaseDropdown" title="Use Cases">
-                                <NavDropdown.Item href="/use-cases/">All Use Cases</NavDropdown.Item>
-                                <NavDropdown.Item href="/use-cases/code-security">Code Security</NavDropdown.Item>
-                                <NavDropdown.Item href="/use-cases/onboarding">Developer Onboarding</NavDropdown.Item>
-                                <NavDropdown.Item href="/use-cases/incident-response">
-                                    Incident Response
-                                </NavDropdown.Item>
-                                <NavDropdown.Item href="/use-cases/code-reuse">Code Reuse</NavDropdown.Item>
-                                <NavDropdown.Item href="/use-cases/code-health">Code Health</NavDropdown.Item>
-                            </NavDropdown>
-
-                            <Nav.Link href="/pricing">Pricing</Nav.Link>
-
-                            <Nav.Link href="https://docs.sourcegraph.com" target="_blank" rel="noreferrer">
-                                Docs
-                            </Nav.Link>
+                            {navLinks.map(navLink =>
+                                navLink.items.length === 1 ? (
+                                    navLink.items.map(item =>
+                                        item.href.includes('http') ? (
+                                            <Nav.Link
+                                                key={camelCase(item.title)}
+                                                href={item.href}
+                                                target="_blank"
+                                                rel="noreferrer"
+                                            >
+                                                {item.title}
+                                            </Nav.Link>
+                                        ) : (
+                                            <Nav.Link key={camelCase(item.title)} href={item.href}>
+                                                {item.title}
+                                            </Nav.Link>
+                                        )
```

Comment:
I think we could make this a reusable rendering function since I see this used a few times. That could reduce our logic even further. You could pass a prop to render as a different element as well.